tests/e2e/local/minikube/install_prereqs_debian.sh

```diff
@@ -63,12 +63,12 @@ fi
 
 # Install minikube.
 function install_minikube() {
-  if ! curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.27.0/minikube-linux-amd64 \
-      && chmod +x minikube \
-      && sudo mv minikube /usr/local/bin/; then
-    echo "Looks like minikube installation failed."
-    echo "Please install it manually and then run this script again."
-    exit 1
+  if ! (curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.27.0/minikube-linux-amd64 && \ 
+      chmod +x minikube && \
```

Comment:
Thanks for review @hklai 

I tried to put && before these lines but it does not work because I got the following errors:

```
./install_prereqs_debian.sh: line 67: syntax error near unexpected token `&&'
./install_prereqs_debian.sh: line 67: `      &&chmod +x minikube \'
```

So, I assume we should keep the && after the command line?

Reply:
Interesting. The old version has && in the beginning as well, so it does not work just because they now sit inside ()?

If that is the case, I think we can just leave it as is.

Reply:
@hklai 

> The old version has && in the beginning as well, so it does not work just because they now sit inside ()?

Yes, exactly. We need to have these command sit inside (), otherwise `! curl -Lo minikube-linux-amd64`  will always return false and the following commands `chmod + minikube && mv minikube /usr/local/bin/` would be skipped.

Would you please approve this PR since it's a bug fix?